spec/examples/actions/ui.spec.js

```diff
@@ -89,28 +89,30 @@ describe('interfaceStateActions', () => {
     });
   });
 
-  describe('applicationErrorTriggered', () => {
+  describe('notificationTriggered', () => {
     beforeEach(() => {
-      store.dispatch(applicationErrorTriggered('some-error'));
+      store.dispatch(notificationTriggered('some-error', 'error'));
     });
 
-    it('sets applicationErrors to contain argument', () => {
+    it('sets notifications to contain argument', () => {
```

Comment:
GC, also a good reminder to add an explicit spec for constructing an action with a `payload`